### DISCUSSION TITLE
Validate Clerk JWT Tokens in ASP.NET Core APIs

### DISCUSSION THREAD

**Discussion by hypertrends** [clerk/javascript#1808] [Support request - DO NOT OPEN HERE]
Hello, We have an ASP.NET Core API and we are authorizing access to this API via Clerk JWT bearer tokens. What steps do we need to take to validate the incoming JWT bearer token? We don't mind writing our own SDK in C# to encapsulate your APIs. We were simply looking to see if there was any guidance on how to validate JWT Tokens. 

Thank you!

**Comment by agis** [comment]
Hello @hypertrends, apologies for the late response. Please refer to https://clerk.com/docs/backend-requests/handling/manual-jwt for instructions on how to manually verify Clerk's session tokens.

Let us know if there are other questions!
